patch.patch

```diff
@@ -0,0 +1,390 @@
+diff --git a/apps/inline-help/package.json b/apps/inline-help/package.json
```

Comment:
We need to delete this file before shipping.

Reply:
Ah yeah, my bad...I ought to be _much more_ careful with things like this.